src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        // 1. Read the data from the input CSV file
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("C:\\Users\\nickt\\Desktop\\reportToRead.csv");

```

Comment:
Avoid hardcode in your solution. Pass the file path as a parameter to the `main` method or use a constant.